runtime/src/bank_forks.rs

```diff
@@ -285,16 +285,21 @@ impl BankForks {
                         .read()
                         .unwrap()
                         .root_slot_deltas();
-                    if let Err(e) =
-                        accounts_background_request_sender.send_snapshot_request(SnapshotRequest {
-                            snapshot_root_bank,
-                            status_cache_slot_deltas,
-                        })
-                    {
-                        warn!(
-                            "Error sending snapshot request for bank: {}, err: {:?}",
-                            root_slot, e
-                        );
+                    if snapshot_root_bank.is_startup_verification_complete() {
```

Comment:
What about moving this `is_startup_verification_complete()` check up to the `if` at line 276? Or, at least before getting the slot deltas on line 283?